### ISSUE TITLE
D10 isn't a dice item

### ISSUE THREAD

**Issue by japjustaplayer** [UnderscoreKilburn/repentance-issues#1139]
D10 used to be rng reliable like any other dice item, unlike how it is now. My solution to this is that its devolving effect works 50% of the time for each enemy with the addition of 25% to evolve enemy instead, and the last 25% is a chance that the item will devolve enemy twice to possibly compensate a risk.

**Comment by Turtier** [comment]
Evolving could cause problems since some evolved forms are way too strong. Also a big reason for the change was to make the item good / useable. I think a way to make it more random while still having it be good overall would maybe give 50% chance to devolve and 50% chance to upgrade an enemy into a champion. Or maybe 33% for each with the last 33% to just reroll it into another random enemy that can appear on the current floor your on. I doubt it'll get changed because it FINALLY got good, and in its current form its like spindown dice, and I'd argue that both are more of a dice than d7 is. But still could be worth looking into ways of adding some rng.

**Comment by japjustaplayer** [comment]
You are absolutely right about evolved forms, I should've realized how unfair that would be. It could be a really fun luck based combat item, rather than just a regular active item serving its purpose. It's weird to find that a d10 dice item isn't based on randomness because I always thought that the best way d10 could be buffed is to just make its effect fair and add reason why you should risk gambling with it.
